plugins/repository-hdfs/build.gradle

```diff
@@ -89,12 +147,66 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
 
 if (fixtureSupported) {
   integTestCluster.dependsOn hdfsFixture
+  integTestRunner.systemProperty 'tests.rest.suite', 'hdfs_repository'
 } else {
   logger.warn("hdfsFixture unsupported, please set HADOOP_HOME and put HADOOP_HOME\\bin in PATH")
   // just tests that the plugin loads
   integTestRunner.systemProperty 'tests.rest.suite', 'hdfs_repository/10_basic'
 }
 
+boolean secureFixtureSupported = false;
+if (fixtureSupported) {
+  // Only do secure fixture support if the regular fixture is supported,
+  // and if vagrant is installed
+  def pipe = new ByteArrayOutputStream()
+  try {
+    def runResult = exec {
+      commandLine 'vagrant', '--version'
```

Comment:
I guess this is fine for now, but I dont' like that we generate vagrant tasks which also check this. I think we should do this earlier, perhaps statically in VagrantCommand or something like that.